go-controller/pkg/ovn/pods.go

```diff
@@ -390,6 +417,33 @@ func (oc *Controller) addLogicalPort(pod *kapi.Pod) (err error) {
 		}
 	}
 
+	// add src-ip routes to GR if external gw annotation is set
+	routingExternalGWs, err := oc.getRoutingExternalGWs(pod.Namespace)
+	if err != nil {
+		return err
+	}
+	if routingExternalGWs != nil {
+		podIP := podIfAddrs[0].IP.String()
+		gr := "GR_" + pod.Spec.NodeName
+		for _, v := range routingExternalGWs {
+			gw := v.String()
+			_, stderr, err := util.RunOVNNbctl("--may-exist", "--policy=src-ip", "--ecmp",
```

Comment:
can we batch all these commands and send them with the golang bindings?

Reply:
Thing is I don't see the bindings supporting ecmp:

https://github.com/eBay/go-ovn/blob/master/logical_router_static_route.go

Reply:
You can batch with the command line as well. Just add `"--"` between each operation...

Reply:
ack, thx